app/javascript/controllers/nav_controller.js

```diff
@@ -0,0 +1,31 @@
+import { Controller } from 'stimulus';
+
+export default class extends Controller {
+  static targets = ['userMenu', 'mobileMenu', 'mobileOpenMenuIcon', 'mobileCloseMenuIcon']
+
+  showOrHideUserMenu() {
```

Comment:
minor comment but this could be named `toggleUserMenuVisibility` and the method below `toggleMobileMenuVisibility`

Reply:
I like it!